rust/otap-dataflow/crates/admin/src/lib.rs

```diff
@@ -75,6 +76,17 @@ pub async fn run(
             details: format!("{e}"),
         })?;
 
+    otel_info!(
```

Comment:
Since the event name is required, could we rewrite this macro to use a more regular syntax (see the proposal below)?

```rust
otel_info!(
  "admin_server_listening",
  endpoint: addr.to_string(),
  message: "Admin HTTP server listening"
);

```